snuba/consumers/consumer_builder.py

```diff
@@ -63,10 +63,10 @@ def __init__(
 
         self.metrics = util.create_metrics(
             dogstatsd_host, dogstatsd_port, 'snuba.consumer',
-            tags=[
-                "group:%s" % group_id,
-                "dataset:%s" % self.dataset_name,
-            ]
+            tags={
```

Comment:
This is not covered. (This entire file has 0% coverage.) Similar risks to the replacer building noted earlier.